### ISSUE TITLE
EventEmitter memory leak warning

### ISSUE THREAD

**Issue by RonMelkhior** [mkozjak/node-telnet-client#81]
Hi,

I keep getting after 10 queries the memory leak warning.

My code:
```js
        this.socket = new Telnet();
        this.socket.connect({
            host,
            port,
            shellPrompt: 'Welcome to the TeamSpeak 3 ServerQuery interface, type "help" for a list of commands and "help <command>" for information on a specific command.',
            irs: "\n\r",
            echoLines: -1,
            timeout: 600000,
        });

        // exec code
        this.socket.exec(command, { shellPrompt: 'error id=' })
```

**Edit:** The warnings:

```
(node:9408) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 bufferexceeded listene
rs added. Use emitter.setMaxListeners() to increase limit
(node:9408) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 responseready listener
s added. Use emitter.setMaxListeners() to increase limit
```

Example server: 91.121.83.227:10011

Cheers

**Comment by mkozjak** [comment]
Thank you for reporting this bug! Should be fixed in version 0.13.3.

**Comment by mkozjak** [comment]
In example:
```js
const telnet = require('telnet-client');

this.socket = new telnet();
this.socket.connect({
    host: hostname,
    port: port,
    shellPrompt: 'Welcome to the TeamSpeak 3 ServerQuery interface, type "help" for a list of commands and "help <command>" for information on a specific command.',
});

this.socket.on('ready', (prompt) => {
  setInterval(() => {
    this.socket.exec('help', {
        shellPrompt: 'error id=0 msg=ok'
    }, (error, response) => {
        if (error)
            throw new Error(error)
    })  
  }, 500)
})

this.socket.on('error', (error) => console.log(error))
this.socket.on('close', () => console.log("closed"))
```
